package.json

```diff
@@ -33,6 +33,7 @@
     "lint": "eslint 'packages/**/*.js'",
     "prerelease": "test -n \"$GH_TOKEN\" || (echo 'GH_TOKEN missing'; exit 1)",
     "release": "lerna publish",
+    "release:preview": "yarn release --conventional-prerelease --preid preview",
```

Comment:
Do we actually plan to release a `next`-version? Or do we rather switch to `alpha` before doing the very first pre-release of Hops 14?

Reply:
I would like to do some nightly releases from master - therefore I added this command.